include/flatbuffers/flatbuffers.h

```diff
@@ -75,6 +75,9 @@
 
 // The wire format uses a little endian encoding (since that's efficient for
 // the common platforms).
+#if defined(__s390x__)
+  #define FLATBUFFERS_LITTLEENDIAN 0
+#endif // __s390x__
 #if !defined(FLATBUFFERS_LITTLEENDIAN)
   #if defined(__GNUC__) || defined(__clang__)
     #ifdef __BIG_ENDIAN__
```

Comment:
With flag __BIG_ENDIAN defined, build failure issues were seen on x86 & Travis build. 
Hence, the platform define(__s390x__) has been added.